javaslang-benchmark/src/test/java/javaslang/benchmark/JmhRunner.java

```diff
@@ -29,20 +29,20 @@ public static void devRunWithAssertions(Class benchmarkClass) {
         runAndReport(benchmarkClass, QUICK_WARMUP_ITERATIONS, QUICK_MEASUREMENT_ITERATIONS, Assertions.Enable);
     }
 
-    private static void runAndReport(Class benchmarkClass, int warmUpIterations, int measurementIterations, Assertions assertions) {
-        Collection<RunResult> results = run(benchmarkClass, warmUpIterations, measurementIterations, assertions);
+    private static void runAndReport(Class benchmarkClass, int warmupIterations, int measurementIterations, Assertions assertions) {
```

Comment:
I have added `Class<?>` now. 

I left it out because it is not needed and doesn't give you anything. A different story would have been if we were using `Class<? extends Benchmark>`. However, I concede `Class<?>` is more accurate as `Class<>` is a parameterized class.